tools/index.d.ts

```diff
@@ -0,0 +1,168 @@
+import { RawSourceMap } from 'source-map';
+
+export type ECMA = 5 | 6 | 7 | 8;
```

Comment:
`terser` is not exported as an ES module or even as something that would be detected as an ES module by babel, webpack or typescript with `--esModuleInterop`. We can't have direct exports.

The proper way to write this definition would be to move _all_ of the declarations, including the `export`s, inside a `namespace UglifyJS {}`, and having `export = UglifyJS` to signify a commonjs export.